crates/ruff_linter/src/rules/ruff/rules/unnecessary_round.rs

```diff
@@ -113,9 +113,10 @@ pub(super) enum RoundedValue {
 /// The type of the second argument to `round()`
 #[derive(Clone, Copy, Debug, Eq, PartialEq)]
 pub(super) enum NdigitsValue {
-    NotGiven,
-    LiteralNone,
-    Int(InferredType),
+    NotGivenOrNone,
+    LiteralNegativeInt,
+    LiteralNonNegativeInt,
```

Comment:
What do you think about `LiteralPositiveInt` instead to avoid the double negative?

Reply:
I'd go with. It simplifies the matches where the negative flag isn't relevant and it avoids the `NotNegative` (which feels like a double negative)

```suggestion
    LiteralInt { negative: bool }
```